README.md

```diff
@@ -331,6 +333,58 @@ Drop data for Cetus bounty missions. [(Source)](https://n8k6e2y6.ssl.hwcdn.net/r
 ]
 ```
 
+### /data/zarimanRewards.json
+
+Drop data for Zariman bounty missions. [(Source)](https://n8k6e2y6.ssl.hwcdn.net/repos/hnfvc0o3jnfvc873njb03enrf56.html#zarimanRewards)
+
+#### Data structure:
+```json
+"zarimanRewards": [
+  {
+    "_id": "255771097c27769a3a76bfa38c9fba49",
+    "bountyLevel": "Level 50 - 55 Zariman Bounty",
+      "rewards": {
+        { ... }
+        "C": [
+          {
+            "_id": "d5ccc328b76d91df416cc361d2209c1f",
+            "itemName": "15,000 Credits",
+            "rarity": "Uncommon",
+            "chance": 17.39,
+            "stage": "Final stage"
+          },
+        { ... }
+      ],
+    }
+  },
+  { ... }
+]
+```
+
+### /data/syndicates.json
+
+Drop data for syndicate vendors. Generated by data from the [Warframe Wikia](https://warframe.fandom.com/wiki/Syndicate).
+
+#### Data structure:
+```json
+{
+  "syndicates": {
+    "Steel Meridian": [
+      {
+        "_id": "d765637f1f1ae09add1b5265f5d29ea1",
+        "item": "Steel Meridian Sigil",
+        "chance": 100,
+        "rarity": "Common",
+        "place": "Steel Meridian (Brave) (1000 standing)",
```

Comment:
treat standing like rotation on most drops:
![image](https://user-images.githubusercontent.com/7128721/213213197-2add6e43-3d13-48a2-9ad6-155f0f3e7f68.png)
`Steel Meridian, Brave` is what i'm looking for

Reply:
Ok, i'll change the format into that.

Would still like to add some information as to which vendor you need to be, for example the `Vermisplicer Blueprint` can be bought at Entrati with rank Neutral at Father.
Would you be ok formatting that like `Entrati (Father), Neutral`?

Reply:
That would be perfectly applicable :+1: